drivers/hmis/app/models/hmis/hud/validators/enrollment_validator.rb

```diff
@@ -62,6 +62,11 @@ def self.validate_entry_date(enrollment, household_members: nil, options: {})
     errors = HmisErrors::Errors.new
     dob = enrollment.client&.dob
     exit_date = enrollment.exit_date
+    project_start_date = enrollment.project.operating_start_date
+    project_end_date = enrollment.project.operating_end_date
+
+    errors.add(:entry_date, :out_of_range, message: before_project_start_message(project_start_date), **options) if project_start_date&.> entry_date
+    errors.add(:entry_date, :out_of_range, message: after_project_end_message(project_end_date), **options) if project_end_date&.< entry_date
```

Comment:
woah `&.< ` didn't know that was a thing

Reply:
It's definitely more concise, hopefully this isn't confusing. Fwiw this works for all(?) operators in ruby (`ary&.[int], count&.+2`) etc.